influxdb/files/influxdb.conf

```diff
@@ -187,7 +187,11 @@ reporting-disabled = false
 [http]
   enabled = true
   bind-address = "{{ server.http.bind.address }}:{{ server.http.bind.port }}"
+  {%- if server.admin_user is defined %}
```

Comment:
There is already an entry named 'admin' but it is to set the admin service (port, host). It is not related to the admin_user. If you prefer I can use adminuser.